### PR TITLE
Resolves some build warnings in the ImportExport project.

### PR DESCRIPTION
**Pull request by valadas**
Resolves some (I did not count but probably around 100) build warnings in the Import Export library.

### CODE REVIEW

File: DNN Platform/Modules/DnnExportImport/Components/Common/Constants.cs [hunk 1]
```diff
@@ -2,61 +2,68 @@
...
+    using System;
+
+    /// <summary>
+    /// Lists constants used across the library.
+    /// </summary>
+    public class Constants
+    {
+        /// <summary>
+        /// This is the currently supported schema version support as of this release.
+        /// In future releases this must be updated to be compatible wiht the schema version.
```
Comment (bdukes) [inline DNN Platform/Modules/DnnExportImport/Components/Common/Constants.cs:17]:
```suggestion
        /// In future releases this must be updated to be compatible with the schema version.
```

File: DNN Platform/Modules/DnnExportImport/Components/Common/Util.cs [hunk 2]
```diff
@@ -2,284 +2,381 @@
...
-    {
-        private static readonly ILog Logger = LoggerSource.Instance.GetLogger(typeof(Util));
-        private static int _noRole = Convert.ToInt32(Globals.glbRoleNothing);
-
-        // some string extension helpers
-        public static bool IsNullOrEmpty(this string s) => string.IsNullOrEmpty(s);
-
-        public static bool IsNullOrWhiteSpace(this string s) => string.IsNullOrWhiteSpace(s);
-
-        public static bool HasValue(this string s) => !string.IsNullOrEmpty(s);
```
Comment (bdukes) [inline DNN Platform/Modules/DnnExportImport/Components/Common/Util.cs:37]: We [ship these functions](https://www.fuget.org/packages/DotNetNuke.SiteExportImport/9.7.1/lib/net45/DotNetNuke.SiteExportImport.dll/Dnn.ExportImport.Components.Common/Util?code=true#M%3ADnn.ExportImport.Components.Common.Util.IsNullOrEmpty%28System.String%29) (this project is [published as a NuGet package](https://www.nuget.org/packages/DotNetNuke.SiteExportImport/9.7.1)), so I don't think we can remove them without deprecating.

**Review by bdukes — APPROVED** [review]
Let's deprecate those methods in `Util`, otherwise this looks good to me

File: DNN Platform/Modules/DnnExportImport/Components/Common/Util.cs (hunk 2 above)
Comment (valadas) [reply DNN Platform/Modules/DnnExportImport/Components/Common/Util.cs:37]: Oh, makes sense, I had no clue we published that on Nuget. I have restored them and deprecated them

**Review by bdukes — APPROVED** [review]

**Review by david-poindexter — APPROVED** [review]
Awesome @valadas - thank you! 🎉

**Pull request merged**